iina/ControlBarView.swift

```diff
@@ -15,7 +15,8 @@ class ControlBarView: NSVisualEffectView {
   var isDragging: Bool = false
 
   override func awakeFromNib() {
-    self.layer?.cornerRadius = 6
+    self.layer?.cornerRadius = 4
+    self.translatesAutoresizingMaskIntoConstraints = true;
```

Comment:
XIBs don't make this property available directly; this would require either using `@IBInspectable` or key path runtime attributes.